BEIMA.Client/src/pages/Users/UserPage.js

```diff
@@ -91,13 +76,26 @@ const UserPage = () => {
       }
 
       // Call Update User
-      console.log(newUser);
-      setEditable(false)
+      let updateResult = await UpdateUser(newUser);
+      if(updateResult.status === Constants.HTTP_SUCCESS){
+        Notifications.success("Update User Successful", `User ${username} updated successfully.`);
+        setEditable(false);
+      } else {
+        Notifications.error("Unable to Update User", `Update of User ${username} failed.`);
+      }
     }
     
-    const deleteUserCall = () => {
-      //TODO integrate with styled error/confirm message
-      alert("you sure?");
+    const deleteUserCall = async () => {
```

Comment:
I think having a check for if the current user is trying to delete themselves would be good. I can update this as I work on the authentication side of things

Reply:
I think we should definitely prompt the user before deleting their own account, but I don't know that we should prevent them from doing so.